packages/react-core/src/demos/Card/Card.md

```diff
@@ -743,6 +745,17 @@ class CardViewBasic extends React.Component {
               ))}
             </Gallery>
           </PageSection>
+          <PageSection isFilled={false} sticky="bottom" padding={{default: "noPadding"}} variant="light">
```

Comment:
Can you add `isFilled` to the section with the cards?

Reply:
good catch